### ISSUE TITLE
0013870: ItemOperations -> Fetch -> Store=Mailbox -> Status 14 ? What store to use to get contacts?

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#6859]
**Reported by Jobisoft on 8 Jun 2018 12:23**

I am trying to retrieve contacts and events identified by their ServerID fom Tine.
I am using the Mailbox store (wich wokrs with Microsoft Servers), but get back Status 14.

Status 14: Mailbox fetch provider - The item failed conversion.

What vale for the store do i have to use to get my contacts and events?

...
<?xml version="1.0"?>
<ItemOperations xmlns='ItemOperatons'>
<Fetch>
<Store>Mailbox</Sore>
<CollectionId xmlns='AirSync'>addressbook-root</CollectionId>
<ServerId xmlns='AirSync'>f4ea6e4f3c1979d1e03eec28c66787151858634c</ServerId>
</Fetch>
</ItemOperations>
...

Response

...
<?xml version="1.0"?>
<ItemOperations xmlns='ItemOperatons'>
<Status>1</Status>
<Response>
<Fetch>
<Status>1</Status>
<CollectionId xmlns='AirSync'>addressbook-root</CollectionId>
<ServerId xmlns='AirSync'>f4ea6e4f3c1979d1e03eec28c66787151858634c</ServerId>
</Fetch>
<Status>14</Status>
</Response>
</ItemOperations>
...

**Comment by pschuele** [comment]
i think this Mailbox store is not working with tine20. do you have more information on the ActiveSync client you are using?

**Comment by jobisoft** [comment]
I am developing my own client for thunderbird:
https://addons.mozilla.org/en-US/thunderbird/addon/tbsync/

On github:
https://github.com/jobisoft/TbSync

I am using ItemOperation.Fetch (which does not work with Tine) and Sync.Fetch as fallback (which does work with Tine). Reporting to make you aware of this issue.

**Comment by pschuele** [comment]
ah, that's you :smile: 
thanks for the additional information.

**Comment by jobisoft** [comment]
Is there another connection between us, I am not aware of? But, yes, it is me :-)

This is not limited to contacts, but also true for events and tasks, or any fetchable object.

**Comment by pschuele** [comment]
> Is there another connection between us, I am not aware of? But, yes, it is me :-)

no, just here (and the mantis tracker) - i think, there is another issue with tbsync. I'll put it in the title, too.

> This is not limited to contacts, but also true for events and tasks, or any fetchable object.

ok, removing the Addressbook label.
